numpyro/distributions/continuous.py

```diff
@@ -1033,6 +1033,40 @@ def icdf(self, q):
         return self.loc + self.scale * logit(q)
 
 
+class LogUniform(TransformedDistribution):
```

Comment:
I think you can rename this to [Reciprocal](https://en.wikipedia.org/wiki/Reciprocal_distribution) if the support is `[low, high]` (rather than `[exp(low), exp(high)]` as in `LogUniform`).

Reply:
My understanding is that Reciprocal and [log-uniform](https://docs.scipy.org/doc/scipy/tutorial/stats/continuous_loguniform.html) are identical. Isn't it ?

Reply:
I'm not sure. In LogNormal, loc and scale are parameters of the base distribution. So probably using LocUniform is confusing. What do you think?

Reply:
I also can find implemantation in [Matlab](https://www.mathworks.com/help/stats/loguniform-distribution.html) define support as `[low, high]` 
I think it's better to follow these.